package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "build": "node ./node_modules/gulp/bin/gulp.js",
     "ensure-images": "node ./node_modules/gulp/bin/gulp.js ensure-images-optimized",
-    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target http://localhost:9000/api/"
+    "serve": "yarn run ensure-images && node ./node_modules/polyserve/bin/polyserve -p 5101 -P h2 --proxy-path api --proxy-target https://areafiftylan.nl/api/"
```

Comment:
Have to change this line back, but added it because local test data lacks proper data.